src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,18 @@
+package core.balls;
+
+public class Ball {
+    private String namecolor;
```

Comment:
```suggestion
    private String color;
```
if you use several words - use camelCase

Reply:
all done